modules/teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(name: 'Unknown', age: 0, parent_permission: true, specialization: 'Unknown')
+    super(name: name, age: age, parent_permission: parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
```

Comment:
- [ ] Similarly please remove the default value assigned to the `@specialization` instance variable at line `6` for previously mentioned reasons.